src/rpcprotocol.cpp

```diff
@@ -287,3 +289,68 @@ UniValue JSONRPCError(int code, const string& message)
     error.push_back(Pair("message", message));
     return error;
 }
+
+/** Username used when cookie authentication is in use (arbitrary, only for
```

Comment:
Making a new file pair e.g. `rpcauthcookie` sounds good to me too. But I don't feel strongly about it either. 
The low-level code from rpcprotocol will be unnecessary with the libevent-based HTTP server (#5677), so this will at least leave something there :)